capstone-app/src/app/modules/history/wiki-results.service.spec.ts

```diff
@@ -7,15 +7,11 @@ const testWikiResult = {
     parse: {
       title: 'Stillwater, Oklahoma',
       text: {
-        '*' : '</h2><p>The north-central region of Oklahoma became part of the United States'
-        + 'with the <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/Louisiana_Purchase\"'
-        + 'title=\"Louisiana Purchase\">Louisiana Purchase</a> in 1803. In 1832, author and traveler'
-        +  '<a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/Washington_Irving\" title=\"Washington'
-        + 'Irving\">Washington Irving</a> provided the first recorded description of the area around'
-        + 'Stillwater in his book <i>A Tour on the Prairies</i>. He wrote of “a glorious prairie spreading'
-        + 'out beneath the golden beams of an autumnal sun. The deep and frequent traces of buffalo,'
-        + 'showed it to be a one of their favorite grazing grounds.”<sup id=\"cite_ref-9\" class=\"reference'
-        +'\"><a href=\"#cite_note-9\">&#91;9&#93;</a></sup></p>'
+        '*' : 'The north-central region of Oklahoma became part of the United States with the Louisiana '
```

Comment:
I am unsure if you want this change to be here. This object should mimic the response of the api which includes the HTML tags.

Reply:
My change now no longer includes html tags.

Reply:
Billy is correct here. Although in the component the tags are removed through the parsing, here we are simulating the response of the API for the service. The Wikimedia API still returns the values as HTML, so this should be a string with the tags

Reply:
Ahh, yes, I'm sorry, that makes sense!